AVInput/AVInput.json

```diff
@@ -604,6 +604,23 @@
                 }
             ]
         },
+	"getHdmiVersion": {
+            "summary": "Gets the maximum hdmi compatibility version supported for the given port",
+            "params": {
+                "type":"object",
+                "properties": {
+                    "portId":{
+                        "$ref": "#/definitions/portId"
+                    },
+                },
+                "required": [
+                    "portId",
+                ]
+            },
+            "result": {
+                "$ref": "#/common/result"
```

Comment:
could you confirm if result common defines the EDID version that your report from thunder API  if not refine result properly

Reply:
Updated the json accordingly Amit, Can you please review